CSC8503/GameTechRenderer.cpp

```diff
@@ -136,8 +139,8 @@ void GameTechRenderer::BuildObjectList() {
 	gameWorld.OperateOnContents(
 		[&](GameObject* o) {
 			if (o->IsActive()) {
-				const RenderObject* g = o->GetRenderObject();
-				if (g) {
+			const RenderObject* g = o->GetRenderObject();
+				if (g && mFrameFrustum.SphereInsideFrustum(o->GetTransform().GetPosition(), o->GetRenderObject()->GetCullSphereRadius())) {
```

Comment:
I prefer declaring a variable and check them in if statements rather then complex logic controls.

And maybe we can change the names of the variable g and o, give them a much understandable names .

```suggestion
//Probably not the best variable name :)
bool isActivable = g && mFrameFrustum.SphereInsideFrustum(o->GetTransform().GetPosition(), o->GetRenderObject()->GetCullSphereRadius());
				if (isActivable ) {
```